tests/testflows/rbac/regression.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+import sys
+from testflows.core import *
+
+append_path(sys.path, "..") 
+
+from helpers.cluster import Cluster
+from helpers.argparser import argparser
+from rbac.requirements import *
+
+issue_12507 = "https://github.com/ClickHouse/ClickHouse/issues/12507"
+issue_12510 = "https://github.com/ClickHouse/ClickHouse/issues/12510"
+issue_12600 = "https://github.com/ClickHouse/ClickHouse/issues/12600"
+
+xfails = {
+    "syntax/show create quota/I show create quota current":
+        [(Fail, "https://github.com/ClickHouse/ClickHouse/issues/12495")],
+    "syntax/create role/I create role that already exists, throws exception":
+        [(Fail, issue_12510)],
+    "syntax/create user/I create user with if not exists, user does exist":
+        [(Fail, issue_12507)],
+    "syntax/create row policy/I create row policy if not exists, policy does exist":
+        [(Fail, issue_12507)],
+    "syntax/create quota/I create quota if not exists, quota does exist":
+        [(Fail, issue_12507)],
+    "syntax/create role/I create role if not exists, role does exist":
+        [(Fail, issue_12507)],
+    "syntax/create settings profile/I create settings profile if not exists, profile does exist":
+        [(Fail, issue_12507)],
```

Comment:
Will the testflows checks in out CI fail after I fix https://github.com/ClickHouse/ClickHouse/issues/12507 ?

Reply:
No, right now these are hard-coded xfails and they will need to be removed once you fix #12507.